hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -50,7 +53,8 @@ def get_index(bucket_name, key):
     return result
 
 
-def get_privacy_group_matcher_active(privacy_group_id: str) -> bool:
+@lru_cache(maxsize=1)
```

Comment:
Note: This will only cache for a single privacy group.

Is this intentional?

Reply:
yeah I was wrong, I changed it to 128, do you think it's good enough?